fuse_ll/src/fuse/mount.rs

```diff
@@ -493,7 +514,7 @@ pub fn umount(mount_point: &Path) -> i32 {
 
 #[cfg(any(target_os = "macos"))]
 pub fn mount(mount_point: &Path, options: &[&str]) -> RawFd {
-    let _args = FuseMountArgs::parse(options);
+    let mut _args = FuseMountArgs::parse(options);
```

Comment:
remove '_',  the original '_' is to suppress unused variable error.

Reply:
Remove underscore _

Reply:
Done.